lib/tool_shed/webapp/config_schema.yml

```diff
@@ -482,6 +482,16 @@ mapping:
           disposable email address during the registration.  If their address domain
           matches any domain in the blacklist, they are refused the registration.
 
+      whitelist_file:
+        type: str
+        default: config/disposable_email_whitelist.conf
```

Comment:
Ok, I will change it. But this is based on the black list feature, which hardcodes config in the same files. Do you want me to change it for blacklist too?

Reply:
I wouldn't worry about blacklist: it's one of the many things we need to change in the config system (unless there is a good reason for keeping that file in galaxy root); also changing it would require gracefully handling existing setups, so it's best done in a separate PR. Also, a `config` prefix would be stripped away here: https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/config/__init__.py#L291